### PR TITLE
LIVY-343. Web UI: Create Session Page

### PR DESCRIPTION
**Pull request by ajbozarth**
[LIVY-343](https://issues.cloudera.org/browse/LIVY-343)

Created a Session Summery Page for Interactive Sessions. Batch Sessions do not have Session Summery Pages because all relevant info is displayed on the All Sessions page Batches table.

The Session Summery Page displays session info (same as in All Sessions table) at the top of the page similar to the Job or Stage Pages in Spark. Below the session info section is a statements table. The statements table contains statement information including statement output. The statement output columns account for empty, error, and data output.

Other new features and updates include:
 - Added a link to the Session Page in the Interactive Sessions table on the All Sessions Page
 - Reorganized static files into a standard directory structure
 - Added bootstrap fonts to fix jQuery DataTables Column headers
 - Added link to the Yarn driver logs to All Sessions tables
 - Updated CSS to use full page width like Spark (was causing issues with statements table)
 - Added a 404 page to the UI

Point of Discussion:
 - Should I change the column order in the Statements table? I played around with it for a while before setting on the current ordering.

Tested manually and passed Livy tests locally.

Screenshots:
1-empty
2-applist
3-session0
4-session1
5-session2
6-session3
7-session4inprog
8-session4
9-session5

### CODE REVIEW

**Comment by ajbozarth** [comment]
@jerryshao finally got this updated to work with all the latest merges. Could you take a look when you have time, its pretty big.

**Comment by jerryshao** [comment]
Thanks @ajbozarth ! I will review the code when I have time.

**Comment by jerryshao** [comment]
@ajbozarth , do we have license issue for the fonts you added?

**Comment by ajbozarth** [comment]
I'm not sure, they're the standard bootstrap fonts so I figured they were okay if the bootstrap html/css/js was, but I can look into that if you're worried

**Comment by jerryshao** [comment]
Yes, please do check it, thanks.

File: server/src/main/scala/org/apache/livy/server/LivyServer.scala [hunk 1]
```diff
@@ -129,7 +129,7 @@ class LivyServer extends Logging {
     val interactiveSessionManager = new InteractiveSessionManager(livyConf, sessionStore)
 
     server = new WebServer(livyConf, host, port)
-    server.context.setResourceBase("src/main/com/cloudera/livy/server")
+    server.context.setResourceBase("src/main/org/apache/livy/server")
```
Comment (jerryshao) [inline server/src/main/scala/org/apache/livy/server/LivyServer.scala]: Thanks for the fix, I forgot to update here.

**Comment by jerryshao** [comment]
LGTM, @ajbozarth can you please trigger the travis test again.

**Comment by ajbozarth** [comment]
I looked into the legal stuff and found that apache/karaf-webconsole uses bootstrap with glyphicons (albeit an older version) and it was approved by legal via [LEGAL-125](https://issues.apache.org/jira/browse/LEGAL-125) so I believe we're good here.

**Comment by jerryshao** [comment]
Thanks, merging to master.

**Pull request closed without merging**